example_dags/README.md

```diff
@@ -0,0 +1,93 @@
+# Running Example DAGs
+
+This directory contains several example Airflow DAGs to demonstrate how to use the
+`SkyPilotClusterOperator`.
+
+To run these examples, you need:
+1. A SkyPilot remote API server (see [Configuration and Usage](../README.md#configuration-and-usage))
+2. An Airflow deployment
+3. An extended Airflow image which has `airflow-provider-skypilot` installed (along with other
```

Comment:
Can mention the actual command for installing the dependency in the image here as well:
```
RUN pip install ...
```